Directory.Packages.props

```diff
@@ -150,15 +150,15 @@
     <PackageVersion Include="Testcontainers.Redis" Version="$(TestcontainersPackageVersion)" />
     <PackageVersion Include="Testcontainers.Nats" Version="$(TestcontainersPackageVersion)" />
     <!-- playground apps dependencies -->
-    <PackageVersion Include="Dapr.AspNetCore" Version="1.13.0" />
-    <PackageVersion Include="Microsoft.Orleans.Clustering.AzureStorage" Version="8.1.0-nightly.20240307.1" />
```

Comment:
is this behind "8.1.0" or ahead?

Reply:
this is going from 8.1.0 unstable, to 8.1.0 stable, so the update is newer (ahead)